starknet-core/src/types/contract/mod.rs

```diff
@@ -1014,6 +1039,7 @@ mod tests {
     struct ContractHashes {
         sierra_class_hash: String,
         compiled_class_hash: String,
+        compiled_class_hash_blake2s: String,
     }
```

Comment:
I know it's only struct in test, but imo it's better to hold separate structs for pre poseidon and blake.